src/components/scenes/GuiPluginListScene.tsx

```diff
@@ -450,8 +452,10 @@ const getStyles = cacheStyles((theme: Theme) => ({
     marginLeft: -theme.rem(0.5),
     width: '100%'
   },
-  rowContainer: {
-    flexDirection: 'column'
+  cardContainer: {
+    flexDirection: 'column',
+    flexShrink: 1,
```

Comment:
What's the flexShrink for? This view is in a ScrollView and wouldn't ever need to shrink

Reply:
The `cardContainer` is the card's contents. I'll rename to clarify further. Without flexShrink it looks like this:
![image](https://github.com/EdgeApp/edge-react-gui/assets/90650827/e7d20c7e-69f1-4377-a7ed-ac619958c320)

It actually needs a right padding as well because on an even narrow device, it looks like this:
![image](https://github.com/EdgeApp/edge-react-gui/assets/90650827/31caad2e-42a8-4f75-bdfd-f477558ca838)